### ISSUE TITLE
Text should be in sufficient contrast ratio.

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8143]
### Describe your problem(s)

Text should be in sufficient contrast ratio.

### Describe your solution

Level AAA
[G17](https://www.w3.org/WAI/WCAG21/Techniques/general/G17.html) - a contrast ratio of at least 7:1 exists

Level AA
[G18](https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html) - a contrast ratio of at least 4.5:1 exists
